pkg/cache/object.go

```diff
@@ -46,7 +47,7 @@ func (o Object) Name() string {
 // CloudFormation template being deployed, e.g. InfrastructureVersion,
 // SpectaVersion.
 func (o Object) Parameter() string {
-	return fmt.Sprintf("%sVersion", cases.Title(language.English).String(o.Name()))
+	return fmt.Sprintf("%sVersion", strings.Map(mapFnc, cases.Title(language.English).String(o.Name())))
```

Comment:
This fixes the version parameter names with dashes and spaces. The title case function can work with dashes out of the box. We just have to strip `-` and ` `.